docs/wiki/[User-Guide]-Advanced-Scenarios.md

```diff
@@ -0,0 +1,77 @@
+<!-- markdownlint-disable first-line-h1 -->
+
+## Scenario 1 - Secure island for seed resources
+
+Depending on your security needs, you may wish to store the Azure resources deployed by the bootstrap in separate subscription and optionally a separate management group hierarchy to the Azure Landing Zone. This could be the case when you need to separate the concerns of deploying and maintaining the Azure Landing Zone from the day to day access of the Azure Landing Zone.
+
+The resources deployed by the bootstrap vary depending on the options you choose, but they may include the following:
+
+- Storage account for state file
+- User assigned managed identities
+- [Optional] Self hosted agents
+- [Optional] Networking, DNS and Private End Point for storage account
+
+In order to use the secure island approach, you can follow these steps:
+
+### Option 1 - Separate subscription under separate management group hierarchy
```

Comment:
Was this decided to be a valid option? I thought we were shying against this due to the lack of governance?

Reply:
As discussed, we will keep this in place for now since it is a theoretical how it can be achieved with the accelerator as opposed to a prescribed best practice.